pkg/autoscaler/aggregation/bucketing.go

```diff
@@ -87,6 +108,49 @@ func roundToNDigits(n int, f float64) float64 {
 	return math.Floor(f*p) / p
 }
 
+const (
+	// minExponent is the minimal decay multiplier for the weighted average computations.
+	minExponent = 0.2
+	// The sum of weights for weighted average should be at least this much.
+	weightPrecision = 0.9999
+	precision       = 6
+)
+
+// WeightedAverage returns the exponential weighted average
```

Comment:
Could we add a couple examples to this comment maybe? e.g. "For example, if the sequence "8, 8, 2, 2" is recorded, the average will be 3 (or whatever) rather than 5, due to recent numbers being more heavily weighted".

Reply:
will do.